### PR TITLE
Added match params as a json which contains "contentFileUri" property

### PR DESCRIPTION
**Pull request by PavelLoparev**

### CODE REVIEW

File: src/Context/Params/MatchContextParameters.php [hunk 1]
```diff
@@ -8,10 +8,11 @@
...
  * @package Context\Params
  */
-class MatchContextParameters extends BaseParameters implements ContextParametersInterface
+class MatchContextParameters extends BaseParameters
 {
 
-    public function setContentFileUri($contentFileUri) {
+    public function setContentFileUri($contentFileUri)
+    {
         $this->params['contentFileUri'] = $contentFileUri;
```
Comment (paveldhq) [inline src/Context/Params/MatchContextParameters.php]: i think `$this->set('contentFileUri', $contentFileUri);` should be used.

File: src/Context/Params/UploadContextParameters.php [hunk 2]
```diff
@@ -27,12 +27,14 @@ public function setContent($contextFileUri)
         $this->params['content'] = $contextFileUri;
     }
 
-    public function setContentFileUri($contentFileUri) {
-        $this->params['contentFileUri'] = $contentFileUri;
+    public function setName($name)
+    {
+        $this->params['name'] = $name;
```
Comment (paveldhq) [inline src/Context/Params/UploadContextParameters.php]: same as above

File: src/Context/Params/UploadContextParameters.php [hunk 3]
```diff
@@ -27,12 +27,14 @@ public function setContent($contextFileUri)
...
     }
 
-    public function setName($name) {
-        $this->params['name'] = $name;
+    public function setMatchParams(MatchContextParameters $params)
+    {
+        $this->params['matchParams'] = json_encode($params->exportToArray());
```
Comment (paveldhq) [inline src/Context/Params/UploadContextParameters.php]: same as above

**Review by paveldhq — CHANGES_REQUESTED** [review]

**Review by dimitrystd — APPROVED** [review]

**Review by alkulikov — COMMENTED** [review]
LGTM

**Pull request merged**